layout/theme.liquid

```diff
@@ -51,6 +51,46 @@
       {{ body_font_bold_italic | font_face: font_display: 'swap' }}
       {{ settings.type_header_font | font_face: font_display: 'swap' }}
 
+      {% for scheme in settings.color_schemes %}
+        {% if forloop.index == 1 %}
+          :root,
+        {% endif %}
+        .color-{{ scheme.id }} {
+          --color-scheme-background: {{ scheme.settings.background.red }},{{ scheme.settings.background.green }},{{ scheme.settings.background.blue }};
+          {% if scheme.settings.background_gradient != empty %}
+            --color-scheme-background-gradient: {{ scheme.settings.background_gradient }};
+          {% else %}
+            --color-scheme-background-gradient: {{ scheme.settings.background }};
+          {% endif %}
+          --color-scheme-text: {{ scheme.settings.text.red }},{{ scheme.settings.text.green }},{{ scheme.settings.text.blue }};
+          --color-scheme-button: {{ scheme.settings.button.red }},{{ scheme.settings.button.green }},{{ scheme.settings.button.blue }};
+          --color-scheme-button-label: {{ scheme.settings.button_label.red }},{{ scheme.settings.button_label.green }},{{ scheme.settings.button_label.blue }};
+          --color-scheme-secondary-button-label: {{ scheme.settings.secondary_button_label.red }},{{ scheme.settings.secondary_button_label.green }},{{ scheme.settings.secondary_button_label.blue }};
+          --color-scheme-shadow: {{ scheme.settings.shadow.red }},{{ scheme.settings.shadow.green }},{{ scheme.settings.shadow.blue }};
+        }
+      {% endfor %}
+
+      {% for scheme in settings.color_schemes %}
+        .color-{{ scheme.id }},
+      {% endfor %}
+      :root
+      {
+        --color-background: var(--color-scheme-background);
+        --gradient-background: var(--color-scheme-background-gradient);
+        --color-foreground: var(--color-scheme-text);
+        --color-shadow: var(--color-scheme-shadow);
+        --color-button: var(--color-scheme-button);
+        --color-button-text: var(--color-scheme-button-label);
+        --color-link: var(--color-scheme-secondary-button-label);
+        --color-badge-foreground: var(--color-foreground);
+        --color-badge-background: var(--color-background);
+        --color-badge-border: var(--color-foreground);
+        --payment-terms-background-color: rgb(var(--color-scheme-background));
+
+        color: rgba(var(--color-foreground), 0.75);
+        background-color: rgb(var(--color-background));
```

Comment:
do we need this in `:root` if it's also applied on `body` below on line `230-231`?

Reply:
Adding to this.. I ended up separating these out like ^ that original version so `:root` isn't also receiving the background-color styles. That's what was causing disprencies between main and my branch.